logback.xml

```diff
@@ -26,8 +26,5 @@
         <logger name="org.kairosdb.core.datastore.KairosDatastore" level="INFO" additivity="false">
                 <appender-ref ref="stdout"/>
         </logger>
-        <logger name="org.kairosdb.datastore.cassandra.CassandraClientImpl" level="DEBUG" additivity="false">
```

Comment:
why was this removed?

Reply:
This was added some months ago to track cassandra client pool status, but appeared to be useless